### PR TITLE
Added tests for message function api

### PR DESCRIPTION
**Pull request by DartingMelody**
## General Checkups
- [x] Have you checked that there aren't other open pull requests for the same issue/update/change?
- [x] If your code includes new features and not just bug fixes/copy editing: Did you include new tests?

### CODE REVIEW

File: ohapi/tests/test_message.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import os
+import vcr
+import unittest
+import ohapi.api as api
+
+my_vcr = vcr.VCR(path_transformer=vcr.VCR.ensure_suffix('.yaml'),
+                 cassette_library_dir='ohapi/cassettes',
+                 filter_query_parameters=['access_token'])
+
+class test_message(unittest.TestCase):
```
Comment (madprime) [inline ohapi/tests/test_message.py]: As with #35 I'd like to have this moved to `test_api.py`, and renamed to match PEP8 naming guidelines for classes, e.g. APITestMessage

**Comment by DartingMelody** [comment]
Hi @gedankenstuecke @madprime, I have renamed and moved the tests for message function. Please review.

File: ohapi/tests/test_api.py [hunk 2]
```diff
@@ -158,3 +167,64 @@ def test_get_page_invalid_access_token(self):
...
+                                  ["Project member IDs are always 8" +
+                                   " digits long."]}}
+
+    @my_vcr.use_cassette()
+    def test_message_all_members_false_project_member_id_not_none_invalid_digit(self):
+        response = message(project_member_ids=[INVALID_PMI1, INVALID_PMI2],
+                           subject=SUBJECT, message=MESSAGE,
+                           access_token=MASTER_ACCESS_TOKEN)
+        assert response.json() == {"errors":{"project_member_ids":
+                                  ["Invalid project member ID(s): invalidPMI2"]}}
```
[the remark this answers is not in the document]
Comment (madprime) [reply ohapi/tests/test_api.py]: Should try to fix this, e.g. maybe like
```
        assert response.json() == {
            "errors": {"project_member_ids": [
                "Invalid project member ID(s): invalidPMI2"]}}
```

File: ohapi/tests/test_api.py [hunk 3]
```diff
@@ -158,3 +167,64 @@ def test_get_page_invalid_access_token(self):
...
+        assert response.json() == {"errors":{"project_member_ids":
+                                  ["Project member IDs are always 8" +
+                                   " digits long."]}}
+
+    @my_vcr.use_cassette()
+    def test_message_all_members_false_project_member_id_not_none_invalid_digit(self):
+        response = message(project_member_ids=[INVALID_PMI1, INVALID_PMI2],
+                           subject=SUBJECT, message=MESSAGE,
+                           access_token=MASTER_ACCESS_TOKEN)
+        assert response.json() == {"errors":{"project_member_ids":
```
[the remark this answers is not in the document]
Comment (madprime) [reply ohapi/tests/test_api.py]: should be `{"errors": {"project_member_ids":`

[the remark this answers is not in the document]
Comment (madprime) [reply ohapi/tests/test_api.py]: Should fix this too (whitespace after colon)

File: ohapi/tests/test_api.py [hunk 4]
```diff
@@ -158,3 +167,64 @@ def test_get_page_invalid_access_token(self):
...
+
+    @my_vcr.use_cassette()
+    def test_message_all_members_true_project_member_id_not_none(self):
+        self.assertRaises(Exception, message, all_members=True,
+                          project_member_ids=['abcdef', 'sdf'],
+                          subject=SUBJECT, message=MESSAGE,
+                          access_token=ACCESS_TOKEN)
+
+    @my_vcr.use_cassette()
+    def test_message_all_members_false_project_member_id_not_none_invalid_char(self):
```
[the remark this answers is not in the document]
Comment (madprime) [reply ohapi/tests/test_api.py]: Is this a test where the project member ID has an invalid character? Maybe `test_message_all_members_false_proj_member_id_has_invalid_char`? (Is that below 79? :) )

**Comment by madprime** [comment]
@DartingMelody awesome! No worries on the variables defined in locals(), but there's a couple other style issues still raised by Hound. It seems minor, but we'd like our code to have a standard style, so I'd like those to be fixed. :)

Have you added a linter into your code editor? If not, you might want to look into this. I use linter-flake8 with atom. It's very useful to be able to see issues immediately, while coding!

**Comment by DartingMelody** [comment]
Hi, @madprime , thanks for suggesting the linter. I switched to atom with flake8 linter. I have fixed the hound issues(except the locals()). Please review. Thanks! :)

**Comment by madprime** [comment]
@DartingMelody Thank you! It's great to have more testing, and I'm glad the tip about a linter was helpful. :) I'll merge now!

**Pull request merged**